Makefile

```diff
@@ -392,4 +392,4 @@ push-docker-images: docker-images
 # For master (until released), major should be 0 and patch should be 0.
 # For release branches, major and minor should be frozen.
 sdk-check-version:
-	go run tools/sdkver/sdkver.go --check-major=0 --check-minor=42
+	go run tools/sdkver/sdkver.go --check-major=0 --check-minor=43
```

Comment:
You changed the makefile which checks for the sdk version freeze
You have to change your SDK version to 0.42.3
same with the SDK_CHANGELOG.md